dbadmin/migrations/002_health_index_null.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE image ALTER COLUMN health_index DROP NOT NULL;
```

Comment:
pls, check how we are doing migrations in this repo,

you need to create two files one is `<name>.up.sql` which upgrades the database (so basically this file)
and second file `<name>.down.sql` which downgrades the database (so it alters the table back to how it was - adds not null constraint to the column